blob/anyblob/anyblob.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package anyblob provides an Open function that opens buckets backed
+// by GCS, S3, Azure Storage, filesystem, and memory.
```

Comment:
Instead of listing them, maybe just say "all known blob providers" or something?